src/test/java/racingcar/service/RaceResultServiceIntegrationTest.java

```diff
@@ -0,0 +1,50 @@
+package racingcar.service;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import racingcar.service.dto.CarStatusResponse;
+import racingcar.service.dto.GameInfoRequest;
+import racingcar.service.dto.RaceResultResponse;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+@DisplayName("RaceResultService Integration Test")
+class RaceResultServiceIntegrationTest {
+
+    @Autowired
+    private RaceResultService raceResultService;
+
+    @Test
+    @DisplayName("createRaceResult() : 게임 정보를 통해 새로운 게임을 만들 수 있다.")
+    void test_createRaceResult() throws Exception {
+        //given
+        final GameInfoRequest gameInfoRequest = new GameInfoRequest("a,b,c,d", 4);
+
+        //when
+        final RaceResultResponse raceResult = raceResultService.createRaceResult(gameInfoRequest);
+
+        //then
+        final List<CarStatusResponse> racingCars = raceResult.getRacingCars();
+
+        assertThat(racingCars).hasSize(4)
+                              .extracting("name")
+                              .containsExactly("a", "b", "c", "d");
```

Comment:
가급적이면 테스트 input과 output에 의존적이지 않게 만들면 좋아요. 대충 아래와 같이?
```suggestion
        final String input = "a,b,c,d";
        final List<String> splitInput = List.of(input.split(","));
        //given
        final GameInfoRequest gameInfoRequest = new GameInfoRequest(input, splitInput.size());

        //when
        final RaceResultResponse raceResult = raceResultService.createRaceResult(gameInfoRequest);

        //then
        final List<CarStatusResponse> racingCars = raceResult.getRacingCars();

        assertThat(racingCars).hasSize(splitInput.size())
                              .extracting("name")
                              .containsAll(splitInput);
```

Reply:
테스트 변경을 최소화하기 위해서 input, output 에 의존적이지 않게 하는걸까요!?

Reply:
네 맞습니다!